_drafts/online-learning-and-its-accessibility.md

```diff
@@ -0,0 +1,16 @@
+# Online learning, and its accessibility in these unprecedented times
```

Comment:
Instead of a level-1 heading for your title, put your title in yaml metadata at the top of your page. To see how to do that, go to the ["Sample Post"](https://github.com/cdl-geneseo/cdl-geneseo.github.io/blob/master/_drafts/sample_post.md) and view the "Raw" text content there. (Find the button at the right.) Your title will automatically be fed into the published page. If you want any section headings within your post, use level-2 (\#\#) headings.